docs/admonitions.rst

```diff
@@ -57,13 +57,33 @@ Version Directives
 ******************
 
 The `versionadded`, `versionchanged`, and `deprecated` directives defined by Sphinx are also
-rendered by the sphinx-immaterial theme as admonitions.
+overridden by the sphinx-immaterial theme to render as :rst:dir:`admonition`\ s with added
```

Comment:
perhaps leave as "rendered by this theme as admonitions, and also optionally overridden to support the additional admonition options collapsible, classes, ...

Reply:
It became a run-on sentence. I used the suggested wording in a additional sentence.